build.sbt

```diff
@@ -37,16 +37,16 @@ riffRaffArtifactResources += (file("cfn.yaml"), s"cloudformation/cfn.yaml")
 
 assembly / assemblyMergeStrategy := {
   case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
-  case x => MergeStrategy.first
+  case _ => MergeStrategy.first
 }
 
 
 buildInfoPackage := "ophan.geoip.extractor"
 buildInfoKeys := Seq[BuildInfoKey](
-  BuildInfoKey.constant("buildNumber", Option(System.getenv("BUILD_NUMBER")) getOrElse "DEV"),
+  "buildNumber" -> Option(System.getenv("BUILD_NUMBER")).getOrElse("DEV"),
   // so this next one is constant to avoid it always recompiling on dev machines.
   // we only really care about build time on teamcity, when a constant based on when
   // it was loaded is just fine
-  BuildInfoKey.constant("buildTime", System.currentTimeMillis),
-  BuildInfoKey.constant("gitCommitId", Option(System.getenv("BUILD_VCS_NUMBER")) getOrElse "DEV")
+  "buildTime" -> System.currentTimeMillis,
+  "gitCommitId"-> Option(System.getenv("BUILD_VCS_NUMBER")).getOrElse("DEV")
```

Comment:
Syntax change here prompted by https://github.com/sbt/sbt-buildinfo/pull/163, which was released with v0.10.0 of the `sbt-buildinfo` plugin.